src/pages/MembersPage/MembersList/MembersList.jsx

```diff
@@ -40,17 +40,18 @@ function useActivateUser({ provider, owner }) {
   return { activate, ...rest }
 }
 
-const handleActivate = (accountDetails, activate, setIsOpen) => (user) => {
-  if (
-    accountDetails?.activatedUserCount >= accountDetails?.plan?.quantity &&
-    !user.activated &&
-    isFreePlan(accountDetails?.plan?.value)
-  ) {
-    setIsOpen(true)
-  } else {
-    activate(user.ownerid, !user.activated)
+const handleActivate =
+  (accountDetails, plan, planUserCount, activate, setIsOpen) => (user) => {
+    if (
+      accountDetails?.activatedUserCount >= planUserCount &&
```

Comment:
I actually just removed accountDetails altogether in the isFreePlan migration PR, wondering if you'd be open to either making the similar change on your end or letting me merge first and rebase

Reply:
I'll remove it